pkgs/by-name/hv/hvm3/package.nix

```
@@ -0,0 +1,38 @@
+{
+  haskell,
+  lib,
+  glibc,
+}:
+
+let
+  inherit (haskell.lib.compose) overrideCabal justStaticExecutables;
+
+  # See HVM3.cabal. GHC2024 >= ghc910.
+  ghcVersion = "ghc910";
+  haskellPackages = haskell.packages.${ghcVersion};
+
+  overrides = rec {
+    pname = "hvm3";
+    version = "${raw-pkg.version}-unstable-2025-03-18";
+
+    postPatch = ''
+      sed -i -E \
+        's/([a-zA-Z0-9_-]+) (\^>=|==)[^,]*,/\1,/g' \
+        HVM3.cabal
+    '';
+
+    # Test suite does nothing.
+    doCheck = false;
+
+    # maintainers = with lib.maintainers; [ joaomoreira ];
```

```suggestion
    maintainers = with lib.maintainers; [ joaomoreira ];
```
Now that the other PR is merged